utils/nctl/sh/scenarios/sync_test.sh

```diff
@@ -56,7 +56,7 @@ function log_step() {
 
 function do_await_genesis_era_to_complete() {
     log_step "awaiting genesis era to complete"
-    while [ "$(get_chain_era)" -lt 1 ]; do
+    while [ "$(get_chain_era)" != "1" ]; do
```

Comment:
Shouldn't that be `0`?

Reply:
Genesis is 0 and we want to transition into the first era after. Recently, code of the client was changed and it now returns `n/a` when it cannot give an answer and the numeric comparison didn't work.